modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
++
+The previous command, `mceInsertTable`, no longer works to invoke an _Insert Table_ dialog box: use the `mceInsertTableDialog` command instead.
++
+The `mceInsertTable` command is still the command for inserting a table with specified properties directly and without asking for user-input.
++
+* The default values for the `table_style_by_css` and the `table_use_colgroups` options have both been changed.
++
+Previously, these options defaulted to `false`. As of {productname} 6.0, both options now default to `true`.
++
+This is in line with modern web practice.
++
+* the `table_responsive_width` option was previously deprecated and superseded by `table_sizing_mode`.
++
+In {productname} 6.0, the `table_responsive_width` option has been removed.
+
+//[[fixed-plugins]]
+//=== Fixed plugins
+//
+//[id-added-plugins]]
+//=== Added plugins
+
+[[removed-or-deprecated-plugins]]
+=== Removed or deprecated plugins
+
+[[bbcode]]
+==== BBCode
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `bbcode` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+[[color-picker]]
+==== Color Picker
+
+Color Picker’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `colorpicker` plugin has been removed.
+
+[[context-menu]]
+==== Context Menu
+
+Context Menu’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `contextmenu` plugin has been removed.
+
+[[full-page]]
+==== Full Page
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `fullpage` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+[[hr]]
+==== HR
+
+The `hr` (horizontal rule) functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `hr` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[changed-plugins-imagetools]]
+==== ImageTools
+
+The `imagetools` plugin is no longer part of the Free {productname} editor. _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+[[legacy-output]]
+==== Legacy Output
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `legacyoutput` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
```

Comment:
```suggestion
As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[{productname} 5 announcement, window="_blank"], the `legacyoutput` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
```